faunadb-java-dsl/src/main/java/com/faunadb/client/types/FaunaConstructor.java

```diff
@@ -0,0 +1,43 @@
+package com.faunadb.client.types;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * <p>Specifies which constructor to use when decoding an object with a {@link Decoder}.
+ * You can also annotate a public static method, which will then be used instead of a constructor.</p>
+ *
+ * <p>This attribute can only be used once per class.</p>
```

Comment:
attribute => annotation

can't remember if i said attribute in my previous comment. if i did, sorry